### DISCUSSION TITLE
Thanks for this great little add-on

### DISCUSSION THREAD

**Discussion by chipwich** [rkusa/DATIS#137] [General]
I've been playing around with DATIS the last couple of days on the latest open beta build of DCS and really like what you can do with this little program.  If you fly IRL, not having ATIS is an annoyance and one of those things that make you wonder why DCS never implemented it.  At first I was a little miffed at having to run an SRS server instance, but it works so well with seemingly low overhead that I don't mind.  Of course this is no big deal running on a dedicated server.

Thanks so much for this very useful program!

-Chipwich

**Comment by rkusa** [comment]
Thank you for the kind words! 😊
